test/helpers/dao.js

```diff
@@ -0,0 +1,62 @@
+const { hash } = require('eth-ens-namehash')
```

Comment:
As a note for the future, we probably want to expose these utilities as part of `contract-helpers`.